### ISSUE TITLE
Ports not being released on stop?

### ISSUE THREAD

**Issue by feichh** [paddybyers/anode#25]
Hi,
It seems to me that the listening sockets are kept open after i stop an isolate, until I force stop the app through the app settings.
This can cause EADDRINUSE errors, when a server is started again.

cheers

**Comment by paddybyers** [comment]
That's correct, and it's because the process itself doesn't exit. Apps that rely on the process exit to clean up will leak, so it is the responsibility of the app to close listeners on exit.

My plan was to extend libuv to track all open handles associated with an isolate, and forcibly close them down. I didn't get to do that, but it's part of the plan for migrating to 0.11.
